### ISSUE TITLE
voila is not compatible with jupyter_server upper than 0.2.0

### ISSUE THREAD

**Issue by bioinfornatics** [voila-dashboards/voila#506]
Dear,
can we update this [line](https://github.com/voila-dashboards/voila/blob/master/setup.py#L379) into the setup.py ?

**Comment by jtpio** [comment]
Yes, the plan is to switch to using the `ExtensionApp` from `jupyter_server`: https://github.com/voila-dashboards/voila/pull/492 (which bumps the dependency to 0.2.0)

**Comment by bioinfornatics** [comment]
thanks I had some tools which needed a recent version

**Comment by stonebig** [comment]
what is the ballpark date  to get a  jupyter_server-0.2.1 compatible version  ?

**Comment by stonebig** [comment]
seems pending this https://github.com/voila-dashboards/voila/pull/492 which is pending jupyter-server-0.3.0 .https://github.com/jupyter/jupyter_server/issues/74 , seems we will have to wait for till Q3 2020 ?
